compiler/rustc_mir/src/util/pretty.rs

```diff
@@ -99,7 +99,9 @@ pub fn dump_enabled<'tcx>(tcx: TyCtxt<'tcx>, pass_name: &str, def_id: DefId) ->
     });
     filters.split('|').any(|or_filter| {
         or_filter.split('&').all(|and_filter| {
-            and_filter == "all" || pass_name.contains(and_filter) || node_path.contains(and_filter)
+            and_filter == "all"
+                || pass_name.contains(and_filter.trim())
+                || node_path.contains(and_filter.trim())
```

Comment:
Could you trim `and_filter` only once and additionally use the trimmed result when comparing against "all"?

Reply:
Absolutely